### ISSUE TITLE
There is a variety of Earth Observation Exploitation Platforms that benefit from the use of interoperable standards both within the platforms and potentially between platforms.

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#110]

**Comment by m-mohr** [comment]
That's a statement, but what's the issue to be solved? (disclaimer: I'm not participating in TCs and don't know the context.)

**Comment by desruisseaux** [comment]
For this comment, I leave the "between platforms" aspect apart. For the "within platform" part, assuming that there is no data transfer involved (i.e. processes are executed where the data is located) do I'm right to assume that we are not talking about web standards like WMS, WCS, etc.? If yes, are we talking about the standards that could shape the API of the languages used by scientists (or other peoples) on those platforms?

**Comment by pebau** [comment]
WMS and WCS, as datacube standards, certainly play a key role. Interop between platforms has been shown in EarthServer by connecting, eg, ECMWF/UK and NCI/AUS.

**Comment by desruisseaux** [comment]
Yes WMS/WCS play a key role in interoperability between platforms, which is why I said in my previous comment that I leave the "between platforms" aspect apart for focussing on the "within platform" part in my question.
